<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AddPerson">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить ученика</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ListOfOrders">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddOrder">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить занятие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_Student">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить ученика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_Order">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить занятие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateReport">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать отчет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Предметы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
